superset/assets/javascripts/dashboard/v2/components/DashboardBuilder.jsx

```diff
@@ -105,15 +108,19 @@ class DashboardBuilder extends React.Component {
               index={0}
               renderTabContent={false}
               onChangeTab={this.handleChangeTab}
+              cells={this.props.cells}
             />
           </WithPopoverMenu>}
 
         <div className="dashboard-content">
           <DashboardGrid
             gridComponent={gridComponent}
             depth={DASHBOARD_ROOT_DEPTH + 1}
+            cells={this.props.cells}
           />
-          {editMode && <BuilderComponentPane />}
+          {this.props.editMode && this.props.showBuilderPane &&
```

Comment:
please destructure `showBuilderPane` with all other props above and use `editMode` and `showBuilderPane` instead of `this.props...`